src/app/pages/LandingPage/components/Promotions/components/PromotionsCarousel/CustomLeftArrow.tsx

```diff
@@ -2,12 +2,13 @@ import React from 'react';
 import { ArrowProps } from 'react-multi-carousel';
 import { faChevronLeft } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { StyledArrow } from './styled';
+import styles from './index.module.scss';
+import classNames from 'classnames';
 
 export const CustomLeftArrow: React.FC<ArrowProps> = ({ onClick }) => {
   return (
-    <StyledArrow isLeft={true} onClick={() => onClick?.()}>
+    <div className={classNames(styles.arrow, 'tw-left-4')} onClick={onClick}>
       <FontAwesomeIcon icon={faChevronLeft} size="2x" />
-    </StyledArrow>
+    </div>
   );
 };
```

Comment:
this can be in the same file with CustomRightArrow.

Reply:
No, it's a separate component so it deserves to be in a separate file (+ I don't see what benefits would it provide).